python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -47,6 +48,22 @@
 cudf.set_option("mode.pandas_compatible", True)
 
 
+def _pandas_util_dir():
+    # In pandas 2.0, pandas.util contains public APIs under
+    # __getattr__ but no __dir__ to find them
+    # https://github.com/pandas-dev/pandas/blob/2.2.x/pandas/util/__init__.py
+    return list(importlib.import_module("pandas.util").__dict__.keys()) + [
```

Comment:
Can this just be
```suggestion
    return dir("pandas.util") + [
```
Then you could also drop importlib.

Reply:
Ah thanks nice shortcut

Reply:
Oh I remember why I didn't use `dir(pandas.util)`, the eventual assignment of this patch to `pd.util.__dir__` will cause a recursion error

Reply:
Ah good point, didn't think of that.